bank-sdk/example-app/src/androidTest/java/net/gini/android/bank/sdk/exampleapp/ui/screens/DigitalInvoiceScreen.kt

```diff
@@ -0,0 +1,213 @@
+package net.gini.android.bank.sdk.exampleapp.ui.screens
+
+import android.widget.TextView
+import androidx.test.espresso.Espresso.onView
+import androidx.test.espresso.action.ViewActions.click
+import androidx.test.espresso.assertion.ViewAssertions.matches
+import androidx.test.espresso.matcher.ViewMatchers.hasChildCount
+import androidx.test.espresso.matcher.ViewMatchers.isDisplayed
+import androidx.test.espresso.matcher.ViewMatchers.withClassName
+import androidx.test.espresso.matcher.ViewMatchers.withId
+import androidx.test.espresso.matcher.ViewMatchers.withText
+import androidx.test.platform.app.InstrumentationRegistry
+import androidx.test.uiautomator.UiCollection
+import androidx.test.uiautomator.UiDevice
+import androidx.test.uiautomator.UiSelector
+import org.hamcrest.Matchers.allOf
+import org.hamcrest.Matchers.`is`
+import org.junit.Assert.assertNotEquals
+
+class DigitalInvoiceScreen {
+    private val device = UiDevice.getInstance(InstrumentationRegistry.getInstrumentation())
+    private var initialValue: String? = null
+    private var updatedValue: String? = null
+
+
+    fun displayDigitalInvoiceTextOnOnboardingScreen(): DigitalInvoiceScreen {
+        val onboardingScreenText = device.findObject(
+            UiSelector()
+                .className("android.widget.TextView")
+                .text("Digital invoice")
+                .resourceId("net.gini.android.bank.sdk.exampleapp:id/onboarding_text_1")
+        )
+        onboardingScreenText.exists()
+        return this
+    }
+
+    fun displayGetStartedButtonOnOnboardingScreen(): DigitalInvoiceScreen {
+        val onboardingScreenButton = device.findObject(
+            UiSelector()
+                .className("android.widget.Button")
+                .text("Get Started")
+                .resourceId("net.gini.android.bank.sdk.exampleapp:id/done_button")
+        )
+        onboardingScreenButton.exists()
+        return this
+    }
+
+    fun clickGetStartedButtonOnOnboardingScreen(): DigitalInvoiceScreen {
+        val onboardingScreenButton = device.findObject(
+            UiSelector()
+                .className("android.widget.Button")
+                .text("Get Started")
+        )
+        onboardingScreenButton.exists()
```

Comment:
do you mean something like this needs to be done?

```
if(onboardingScreenButton.exists()==true) {
            onboardingScreenButton.click()
        }
```

Reply:
the following changes are implemented.

```
if (onboardingScreenButton.isClickable && onboardingScreenButton.exists()) {
            onboardingScreenButton.click()
        }
```

Reply:
here the order should be changed - first check if it exists, otherwise it will throw an exception if the button does not exist and `isClickable` is called